autoload/gutentags/ctags.vim

```diff
@@ -101,7 +101,7 @@ function! gutentags#ctags#generate(proj_dir, tags_file, write_mode) abort
             let l:cmd .= ' -o "' . l:proj_options_file . '"'
         endif
         for ign in split(&wildignore, ',')
-            let l:cmd .= ' -x ' . '"' . ign . '"'
+            let l:cmd .= ' -x ' . '"' . s:escape_patterns(ign) . '"'
```

Comment:
Could this be replaced with `escape('%#')`? (the built-in `escape` function adds backslashes to the provided characters)